scripts/build-xcframework.sh

```diff
@@ -13,14 +13,19 @@ generate_xcframework() {
     local scheme="$1"
     local sufix="${2:-}"
     local MACH_O_TYPE="${3-mh_dylib}"
-    
     local createxcframework="xcodebuild -create-xcframework "
+    local GCC_GENERATE_DEBUGGING_SYMBOLS="YES"
+    
+    if [ "$MACH_O_TYPE" = "staticlib" ]; then
+        #For static framework we disabled everything related to symbols
```

Comment:
`l`: Please elaborate a bit on why we do that.

Reply:
```suggestion
        #For static framework we should disabled symbols because they are not distributed in the framework causing warnings.
```